_faqs/faq-12.md

```diff
@@ -1,8 +1,27 @@
 ---
-question: I want to cite RIOT. Which reference should I use?
+question: How does RIOT compare to XYZ?
 ---
 
-Please consider the following references for citation.
+Compared to **vendor specific solutions**, RIOT runs on many different CPU architectures by many different vendors, from a tiny 8-bit MCU with 2 KiB RAM and 32 KiB flash all the way up to high performance ARM Cortex M and RISC-V MCUs.
+Because RIOT abstracts the hardware differences by providing the same API for the same functionality across this large range of MCUs, projects building upon RIOT are highly portable and therefore resilient to changes in the supply chain.
 
-  * Emmanuel Baccelli, Cenk G&uuml;ndo&#287;an, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdfs/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
+Compared to **Arduino**, RIOT provides advanced features such as native multi-threading and a mature network stack, while still providing a gentle learning curve.
+This allows new users to quickly get productive with RIOT without limiting developers to specific design patterns and software architectures.
+As a nice treat for users migrating from Arduino, our [Arduino compatibility layer](https://doc.riot-os.org/group__sys__arduino.html) allows re-using sketches and libraries from the Arduino world.
 
+Compared to **zephyr**, RIOT is less complex and has a smaller code base.
```

Comment:
```suggestion
Compared to **Zephyr**, RIOT is less complex and has a smaller code base.
```

see https://zephyrproject.org/

We may also want to add links to those alternatives?